resources/views/users/admin/admin_includes/membership.blade.php

```diff
@@ -26,6 +34,16 @@ class="list-group-item">
                     Preview membership card
                 </a>
 
+                @else
+                    @include('website.layouts.macros.confirm-modal', [
+                           'action' => route("user::member::removeend", ['id'=>$user->id]),
+                           'method' => 'POST',
+                           'classes' => 'list-group-item text-danger',
+                           'text' => 'Remove membership end!',
```

Comment:
I think this is a confusing button text. Maybe "Cancel membership removal" would be a better phrasing.